compose-profiles/docker-compose-definitions.yml

```diff
@@ -136,11 +137,11 @@ services:
       - LOG_LEVEL=3
       - UNLYNX_GROUP_FILE_PATH=/medco-configuration/group.toml
       - UNLYNX_GROUP_FILE_IDX=0
-      - DB_HOST=localhost
-      - DB_PORT=5432
-      - DB_NAME=i2b2medco
-      - DB_USER=i2b2
-      - DB_PASSWORD=i2b2
+      - I2B2_DB_HOST=localhost
+      - I2B2_DB_PORT=5432
+      - I2B2_DB_NAME=i2b2medcosrv
+      - I2B2_DB_USER=i2b2
+      - I2B2_DB_PASSWORD=i2b2
```

Comment:
This definition here should have as well the variables `GA_DB_XXX`. This file docker-compose-definitions.yml serves as reference and should define defaults for all the possible variables.